projects/pgai/db/sql/idempotent/012-vectorizer-api.sql

```diff
@@ -280,19 +308,34 @@ set search_path to pg_catalog, pg_temp
 
 -------------------------------------------------------------------------------
 -- disable_vectorizer_schedule
-create or replace function ai.disable_vectorizer_schedule(vectorizer_id pg_catalog.int4) returns void
+create or replace function ai.disable_vectorizer_schedule
```

Comment:
so there are two approaches:
1) ai.disable_vectorizer_schedule(vectorizer_id int4, name text)
2) ai.disable_vectorizer_schedule(vectorizer_id int4) ai.disable_vectorizer_schedule(name text) (overwridding)

The advantage of 2 is that 
ai.disable_vectorizer_schedule('my_name') would work. Right now you'd have to do ai.disable_vectorizer_schedule(name=>'my_name'). 

so I think I'd prefer 2. WDYT?

Reply:
I prefer the override approach (2) as well. You're either using the id OR the name to specify, never both.
What would happen if someone specified the id from one vectorizer and the name of a different one?